docs/command-line-tool.md

```diff
@@ -86,18 +86,18 @@ Lists the active hooks in the current repository along with their state. If `typ
 Manages the shared hook repositories set either globally, or locally within the repository.
 
 ```shell
-$ git hooks shared [add|remove] [--global|--local] <git-url>
-$ git hooks shared clear [--global|--local|--all]
+$ git hooks shared [add|remove] [--shared|--local|--global] <git-url>
+$ git hooks shared clear [--shared|--local|--global|--all]
 $ git hooks shared purge
-$ git hooks shared list [--global|--local|--all] [--with-url]
+$ git hooks shared list [--shared|--local|--global|--all]
 $ git hooks shared [update|pull]
 ```
 
-The `add` or `remove` subcommands adds or removes an item, given as `git-url` from the list. If `--global` is given, then the `githooks.shared` global Git configuration is modified, or if the `--local` option (default) is set, the `.githooks/.shared` file is modified in the local repository.
+The `add` or `remove` subcommands adds or removes an item, given as `git-url` from the list. If `--global` is given, then the `githooks.shared` global Git configuration variables are modified, or if the `--local` option (default) is set, the `.githooks/.shared` file is modified in the local repository.
```

Comment:
This is now changed, right?
Where it says `--local` that is now `--shared` ?

Reply:
Correct!

Reply:
Can you please fix it up?

Reply:
Thx

Reply:
Fixed in lates commit